src/test/scala/com/sky/kafka/configurator/CliParserSpec.scala

```diff
@@ -7,19 +7,20 @@ import common.BaseSpec
 
 class CliParserSpec extends BaseSpec {
 
+  val bootstraServers = "kafka1:2181,kafka2:2195"
```

Comment:
minor typo!

The naming convention in Scala for constants is to make them upper camel case, as explained [here](https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods)